### PR TITLE
fix: Feedback Addressed by @Marcony1 about adding supported Python ve…

### LINKED ISSUE #90: A badge for supported python version
Need to add a badge to indicate the supported Python version as mandated by the package review template.

Also, pointed out by reviewers in https://github.com/UBC-MDS/software-review-2024/issues/18.

### PR DESCRIPTION
**Pull request by Marcony1**
…rsion badge - suggested by @ella-irene @wqxxzd @riyaeliza123 and @charlesxch

### CODE REVIEW

**Comment by SoloSynth1** [comment]
This closes #90.

**Review by atabak-alishiri — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 [![Project Status: Active – The project has reached a stable, usable state and is being actively developed.](https://www.repostatus.org/badges/latest/active.svg)](https://www.repostatus.org/#active)
 [![Documentation Status](https://readthedocs.org/projects/visumorph/badge/?version=latest)](https://visumorph.readthedocs.io/en/latest/?badge=latest)
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+[![Python 3.12.0](https://img.shields.io/badge/python-3.7+-blue.svg)](https://www.python.org/downloads/)
```
Comment (SoloSynth1) [inline README.md:10]: The alt text reads `3.12.0` but the badge is actually `3.7+`.

Also, it does not match our `pyproject.toml`, which states the supported version is from `3.9` and up. Please either change this badge, or change the pyproject.toml.

**Review by SoloSynth1 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SoloSynth1) [reply README.md:10]: @Marcony1

**Pull request merged**